launcher/sdw_updater_gui/Updater.py

```diff
@@ -425,6 +425,63 @@ def _safely_start_vm(vm):
         sdlog.error(str(e))
 
 
+def should_launch_updater(interval):
+    sdlog.info("Starting SecureDrop Launcher")
+
+    status = read_dom0_update_flag_from_disk(with_timestamp=True)
+
+    if _valid_status(status):
+        if _interval_expired(interval, status):
+            sdlog.info("Update interval expired: launching updater.")
+            return True
+        else:
+            if _status_ok_or_rebooted(status):
+                sdlog.info("Update interval not expired, launching client.")
+                return False
+            else:
+                sdlog.info("Updates or reboot required, launching updater.")
```

Comment:
As shown in the test plan, conditions can lead to "Updates or reboot required, launching updater". It might be helpful (to admins & support) to log the actual status code, and possibly timestamp.

Reply:
logger is configured to prepend a timestamp automatically. Added status code in message.